assembly/build.gradle

```diff
@@ -1,121 +1,43 @@
-apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'java'
 
 description = "Sparkling Water Assembly"
 
 dependencies {
-    // Sparkling Water Core
+    // Sparkling Water libraries
+    compile project(":sparkling-water-ml")
+    compile project(":sparkling-water-repl")
     compile(project(":sparkling-water-core")) {
         exclude group: "javax.servlet", module: "servlet-api"
```

Comment:
This was here before, but just an idea -> can we move this exclude to the `shadowRuntime` configuration?

Reply:
Great idea!